src/test/java/com/example/project_spring_boot/controller/ItemControllerTest.java

```diff
@@ -4,6 +4,7 @@
 import com.example.project_spring_boot.entity.Item;
 import com.example.project_spring_boot.service.ItemService;
 import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
It looks as this is never used?

Reply:
Good spot! I removed the ObjectMapper import 👍